src/App.tsx

```diff
@@ -5,7 +5,14 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const addMovie = (newMovie: Movie) => {
+    const exists = (movie: Movie) => movie.imdbId === newMovie.imdbId;
```

Comment:
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig
check variable naming